### PR TITLE
Add `New Featured Artist: t+pazolite` news post

### PR DESCRIPTION
**Pull request by pishifat**
!!!!!!!!!!!!!!!

### CODE REVIEW

File: news/2023/2023-12-23-new-featured-artist-tpazolite.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+Here's one more [Loved osu!taiko map](https://osu.ppy.sh/beatmapsets/639704) hosted by [katacheh](https://osu.ppy.sh/users/6651672)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/396/KAKATTEKOYEAH%E2%80%8B!%E2%80%8B!%E2%80%8B!%E2%80%8B!/t+pazolite feat. Nanahira - Otsukimi PARTY HARD.mp3">
+</audio>
+
+### t+pazolite - Party in the HOLLOWood feat. Nanahira
+
+Take a stab at this [hybrid osu! + osu!taiko set hosted](https://osu.ppy.sh/beatmapsets/819349) hosted by [Annabel](https://osu.ppy.sh/users/3388410)
```
Comment (Walavouchey) [inline news/2023/2023-12-23-new-featured-artist-tpazolite.md:141]:
```suggestion
Take a stab at this [hybrid osu! + osu!taiko set](https://osu.ppy.sh/beatmapsets/819349) hosted by [Annabel](https://osu.ppy.sh/users/3388410)
```

File: news/2023/2023-12-23-new-featured-artist-tpazolite.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    <source src="https://assets.ppy.sh/artists/396/Don_t Waste Me! EP/C-Show - sideflip (tpz Despair Remix).mp3">
+</audio>
+
+---
+
+Your fingers must be tired after scrolling past all of those maps. If you're looking for a more condensed view of these songs (plus a bunch more), [**t+pazolite**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/396) is the place to be.
+
+## Featured Artists in 2024
+
+And with this release, we'll be closing out another year of Featured Artists!
```
Comment (Hiviexd) [inline news/2023/2023-12-23-new-featured-artist-tpazolite.md:336]: mayb also mention how many FA were added in 2023? would be a cool stat

File: news/2023/2023-12-23-new-featured-artist-tpazolite.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+
+Attempt [this osu!taiko map](https://osu.ppy.sh/beatmapsets/237272) hosted by [tasuke912](https://osu.ppy.sh/users/2774767)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/396/The Best of tpz 2 the Best!!!/t+pazolite - CANDY CRACK CURTAIN CALL.mp3">
+</audio>
+
+### t+pazolite - chipscape
+
+Check out [this 12 year old osu! + osu!taiko map](https://osu.ppy.sh/beatmapsets/27752) hosted by [Shiirn](https://osu.ppy.sh/users/465126)!
```
Comment (Walavouchey) [inline news/2023/2023-12-23-new-featured-artist-tpazolite.md:192]:
```suggestion
Check out [this 12-year-old osu! + osu!taiko map](https://osu.ppy.sh/beatmapsets/27752) hosted by [Shiirn](https://osu.ppy.sh/users/465126)!
```

File: news/2023/2023-12-23-new-featured-artist-tpazolite.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+
+## Featured Artists in 2024
+
+And with this release, we'll be closing out another year of Featured Artists!
+
+osu!'s long-term goal with the Featured Artist program has been to bring talent (and their music) to the osu! audience, especially focusing on songs that have history in the osu! community and ensuring they can stick around forever.
+
+We'll continue doing that of course, but throughout the coming year, we also want to bring attention to two other areas:
+
+- **the continuation of our World Cups' [osu! originals](/wiki/osu%21_originals)**, which have already received a ton of attention in 2023 already thanks to [mangomizer](https://osu.ppy.sh/users/1893718)
```
Comment (Walavouchey) [inline news/2023/2023-12-23-new-featured-artist-tpazolite.md:342]:
```suggestion
- **The continuation of our World Cups' [osu! originals](/wiki/osu%21_originals)**, which have already received a ton of attention in 2023 already thanks to [mangomizer](https://osu.ppy.sh/users/1893718)
```

File: news/2023/2023-12-23-new-featured-artist-tpazolite.md [hunk 5]
```diff
@@ -0,0 +1,347 @@
...
+- **artists who aren't well-established on osu!**. Our backlog is already packed with smaller artists who are excited to make their osu! debuts, so stay tuned for a new year of even newer Featured Artists!
```
Comment (Walavouchey) [inline news/2023/2023-12-23-new-featured-artist-tpazolite.md:343]:
```suggestion
- **Artists who aren't well established on osu!.** Our backlog is already packed with smaller artists who are excited to make their osu! debuts, so stay tuned for a new year of even newer Featured Artists!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**